outline/getting-started.md

```diff
@@ -17,30 +16,24 @@ level tour of necessary knowledge for being productive with Tokio.
 A "Hello world!" Tokio tutorial. Implement a server that writes "hello world" to
 the socket.
 
-<a name="runtime-model"></a>
-## Runtime model
-
-**Status**: [Complete](https://tokio.rs/docs/getting-started/runtime-model/)
-
-A crash course into how Tokio executes code.
-
 <a name="futures"></a>
 ## Futures
 
-**Status**: [Complete](https://tokio.rs/docs/getting-started/futures/)
+**Status**: unassigned.
 
-A crash course into the Future abstraction and how Tokio uses it.
+A high level overview of how and when execution happens in Tokio, how futures
+are lazy and the basics of the polling model.
 
-<a name="tasks"></a>
-## Tasks
+<a name="errors"></a>
+## Error Handling
 
-**Status**: [Complete](https://tokio.rs/docs/getting-started/tasks/)
+**Status**: unassigned
 
-Crash course into tasks.
+The very basics of error handling
 
 <a name="chat-server"></a>
-## Tasks
+## Example: Chat Server
```

Comment:
I still think we need to rethink the example here, but I'll leave that for another PR.